corehq/apps/repeaters/models.py

```diff
@@ -315,6 +318,16 @@ def handle_exception(self, exception, repeat_record):
         generator = self.get_payload_generator(self.format_or_default_format())
         return generator.handle_exception(exception, repeat_record)
 
+    @property
+    def is_form_repeater(self):
```

Comment:
Don't think we can do that here. Not sure if there is better way for this since all objects are fetched as instance of `Repeater` class and the doc_type would be the Repeater which won't always be FormRepeater or CaseRepeater but a class inherited from either of those.